models/metrics/metrics.yml

```
@@ -0,0 +1,31 @@
+
+version: 2
+
+metrics:
+  - name: revenue
+    label: Revenue
+    model: ref('orders_by_nation')
+    description: "Income from all orders less tax"
+
+    type: sum
+    sql: net_item_sales_amount
```

```suggestion
    expression: net_item_sales_amount
```
dbt-core v1.3 renamed attributes for metrics:
  'sql'              -> 'expression'
  'type'             -> 'calculation_method'
  'type: expression' -> 'calculation_method: derived'
The old metric parameter names will be fully deprecated in v1.4.